patches/corefx/0003-Enable-CoreFX-tests.patch

```diff
@@ -0,0 +1,388 @@
+deleted file mode 100644
+index 7b665f0..0000000
+--- a/src/System.Runtime.Serialization.Formatters/tests/System.Runtime.Serialization.Formatters.Tests.csproj
++++ /dev/null
+@@ -1,39 +0,0 @@
+-﻿<Project Sdk="Microsoft.NET.Sdk">
+-  <PropertyGroup>
+-    <ProjectGuid>{13CE5E71-D373-4EA6-B3CB-166FF089A42A}</ProjectGuid>
+-    <SkipIncludeNewtonsoftJson>true</SkipIncludeNewtonsoftJson>
+-    <TestRuntime>true</TestRuntime>
+-    <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+-    <Configurations>netcoreapp-Debug;netcoreapp-Release;netfx-Debug;netfx-Release;uap-Debug;uap-Release</Configurations>
+-  </PropertyGroup>
+-  <ItemGroup>
+-    <Compile Include="BinaryFormatterTestData.cs" />
+-    <Compile Include="BinaryFormatterTests.cs" />
+-    <Compile Include="EqualityExtensions.cs" />
+-    <Compile Include="OptionalFieldAttributeTests.cs" />
+-    <Compile Include="FormatterConverterTests.cs" />
+-    <Compile Include="FormatterServicesTests.cs" />
+-    <Compile Include="FormatterServicesTests.Windows.cs" Condition="'$(TargetsWindows)' == 'true'" />
+-    <Compile Include="FormatterTests.cs" />
+-    <Compile Include="PlatformExtensions.cs" />
+-    <Compile Include="SerializationBinderTests.cs" />
+-    <Compile Include="SerializationGuardTests.cs" Condition="'$(TargetsNetCoreApp)' == 'true'" />
+-    <Compile Include="SerializationInfoTests.cs" />
+-    <Compile Include="SerializationTypes.cs" />
+-    <Compile Include="SurrogateSelectorTests.cs" />
+-    <Compile Include="TargetFrameworkMoniker.cs" />
+-    <Compile Include="TypeSerializableValue.cs" />
+-    <Compile Include="$(CommonTestPath)\System\NonRuntimeType.cs">
+-      <Link>Common\System\NonRuntimeType.cs</Link>
+-    </Compile>
+-    <Compile Include="$(CommonTestPath)\System\Runtime\Serialization\Formatters\BinaryFormatterHelpers.cs">
+-      <Link>Common\System\Runtime\Serialization\Formatters\BinaryFormatterHelpers.cs</Link>
+-    </Compile>
+-    <Compile Include="$(CommonTestPath)\System\ThreadCultureChange.cs">
+-      <Link>Common\System\ThreadCultureChange.cs</Link>
+-    </Compile>
+-  </ItemGroup>
+-  <ItemGroup Condition="'$(TargetsNetCoreApp)' == 'true'">
+-    <Reference Include="System.Text.Json" />
+-  </ItemGroup>
+-</Project>
+\ No newline at end of file
+diff --git a/src/dirs.proj b/src/dirs.proj
+index af8d308..30932b8 100644
+--- a/src/dirs.proj
++++ b/src/dirs.proj
+@@ -13,7 +13,7 @@
+     <Project Include="src.builds" />
+     <Project Include="shims\manual\*.csproj" />
+     <Project Include="shims\ApiCompat.proj" Condition="'$(DotNetBuildFromSource)' != 'true'" />
+-    <Project Include="pretest.proj" Condition="'$(DotNetBuildFromSource)' != 'true' and '$(BuildAllConfigurations)' != 'true'" />
++    <Project Include="pretest.proj" Condition="('$(DotNetBuildFromSource)' != 'true' and '$(BuildAllConfigurations)' != 'true') or '$(IncludeTestUtils)' == 'true'" />
```

Comment:
Instead of doing this why not just manually build it after you build binaries?  Alternatively, consider removing the BuildFromSource condition.  /cc @ViktorHofer

Reply:
We didn't want test binaries in non-test source-build builds. AFAIK that's the reason we wanted to have a flag just to enable test utils in separate source-build test builds. cc @crummel

Reply:
Yeah, we generally see the test builds bring in more prebuilts than we would like.  Once our prebuilt process is improved that might the better way to go about this though.